src/PowerShell/Microsoft.WinGet.Client.Engine/Helpers/AppxModuleHelper.cs

```diff
@@ -384,24 +411,30 @@ private async Task InstallVCLibsDependenciesAsync()
                         string? architectureString = psobject?.Architecture?.ToString();
                         if (architectureString == null)
                         {
-                            this.pwshCmdlet.Write(StreamType.Verbose, $"VCLibs dependency has no architecture value: {psobject?.PackageFullName ?? "<null>"}");
+                            this.pwshCmdlet.Write(StreamType.Verbose, $"{packageName} dependency has no architecture value: {psobject?.PackageFullName ?? "<null>"}");
                             continue;
                         }
 
                         architectureString = architectureString.ToLower();
 
-                        if (vcLibsDependencies.ContainsKey(architectureString))
+                        if (dependencies.ContainsKey(architectureString))
                         {
-                            this.pwshCmdlet.Write(StreamType.Verbose, $"VCLibs {architectureString} dependency satisfied by: {psobject?.PackageFullName ?? "<null>"}");
-                            vcLibsDependencies.Remove(architectureString);
+                            this.pwshCmdlet.Write(StreamType.Verbose, $"{packageName} {architectureString} dependency satisfied by: {psobject?.PackageFullName ?? "<null>"}");
+                            dependencies.Remove(architectureString);
                         }
                     }
                     else
                     {
-                        this.pwshCmdlet.Write(StreamType.Verbose, $"VCLibs is lower than minimum required version [{minimumVersion}]: {psobject?.PackageFullName ?? "<null>"}");
+                        this.pwshCmdlet.Write(StreamType.Verbose, $"{packageName} is lower than minimum required version [{minimumVersion}]: {psobject?.PackageFullName ?? "<null>"}");
                     }
                 }
             }
+        }
+
+        private async Task InstallVCLibsDependenciesFromUriAsync()
+        {
+            Dictionary<string, string> vcLibsDependencies = this.GetVCLibsDependencies();
+            this.FindMissingDependencies(vcLibsDependencies, VCLibsUWPDesktop, VCLibsUWPDesktopVersion);

```

Comment:
Update the value of `VCLibsUWPDesktopVersion` to be the one that 1.8 requires (33321?).